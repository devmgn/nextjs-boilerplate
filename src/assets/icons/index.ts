import Help from './help.svg?inline';
import Info from './info.svg?inline';

export { Help, Info };
