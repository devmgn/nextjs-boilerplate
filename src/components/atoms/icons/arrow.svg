<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
<path d="M4 13L4 11L16 11L10.5 5.5L11.92 4.08L19.84 12L11.92 19.92L10.5 18.5L16 13L4 13Z" fill="black"/>
</svg>
