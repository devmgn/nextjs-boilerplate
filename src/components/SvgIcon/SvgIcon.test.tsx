import { render } from "@testing-library/react";
import { SvgIcon } from ".";
import { Close } from "../../assets/icons";

describe("SvgIcon", () => {
  it("renders correctly", () => {
    const { asFragment } = render(<SvgIcon component={Close} />);
    expect(asFragment()).toMatchSnapshot();
  });
});
