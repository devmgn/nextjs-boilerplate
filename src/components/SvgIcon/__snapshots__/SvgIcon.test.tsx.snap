// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`SvgIcon > renders correctly 1`] = `
<DocumentFragment>
  <svg
    aria-hidden="true"
    class="size-[1em] fill-current text-current text-[24px]"
    focusable="false"
  />
</DocumentFragment>
`;
